prisma-fmt/src/get_datamodel.rs

```diff
@@ -190,7 +190,50 @@ mod tests {
                   "isGenerated": false
                 }
               ],
-              "types": []
+              "types": [],
+              "indexes": [
```

Comment:
Would be nice to have a snapshot with `algorithm` and `clustered` properties too.

Reply:
And other properties of `IndexField` too

Reply:
There are actually! But in `dmmf` tests, not `prisma-fmt`.

Reply:
See the new tests in `query-engine/dmmf/test_files`.

Reply:
I can also add something here as well if you'd prefer.